plugins/include/amxmisc.inc

```diff
@@ -921,6 +921,7 @@ stock precache_player_model(const name[], &id = 0)
 	if(file_exists(model))
 		id = precache_generic(model);
 
-	replace_string(model[strlen(model) - 5], charsmax(model), "T.mdl", ".mdl");
+	static const extension[] = "T.mdl";
+	replace_string(model[charsmax(extension)], charsmax(model), extension, ".mdl");
```

Comment:
`charsmax` isn't named good for this case, it would be better to have something like `const_strlen`.